javascript/javascript3/week1/js3-week1.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
You can use a good name for the header of the page. Fx.
```suggestion
    <title>Weather Application</title>
```